libft/sources/strings/strings_internal.h

```diff
@@ -15,36 +15,16 @@
 
 # include <stddef.h>
 # include <stdlib.h>
+# include <stdbool.h>
 # include "strings.h"
 # include "../memory/memory.h"
 
-typedef struct s_delim
-{
-	char	*delim;
-}	t_delim;
-
-typedef struct s_delims
-{
-	t_delim		*delims;
-	int			error;
```

Comment:
From engineering point of view, this field shouldn't be there, because it's not related to delimiter array, it's related to function's control flow... we can leave it like this, but next time maybe we should discuss function signature design deeper.